packages/cli/src/WorkflowHelpers.ts

```diff
@@ -373,15 +374,21 @@ export async function replaceInvalidCredentials(workflow: WorkflowEntity): Promi
  * Returns all IDs if user is global owner (see `whereClause`)
  */
 export async function getSharedWorkflowIds(user: User, roles?: string[]): Promise<string[]> {
+	let allRoles: Role[] = [];
+	const where = { userId: user.id };
+	if (roles) {
+		allRoles = await Db.collections.Role.find({
+			select: ['id', 'name'],
+			where: { name: In(roles) },
```

Comment:
I think we need to add the `scope` as well just to be on the safe side

Reply:
limiting it to workflow and global